src/utils/loggers.js

```diff
@@ -25,7 +25,7 @@ export class WinstonConsoleLogger extends Logger {
     this.logger = createLogger({
       format: format.combine(
         format.timestamp(),
-        format.json(),
+        format.prettyPrint(),
```

Comment:
I think we actually want this to be printed on a single line

Reply:
done 👍